src/generate.ts

```diff
@@ -34,7 +34,37 @@ const colors = {
   black: { skin: ["#ad6453", "#74453d", "#5c3937"], hair: ["#272421"] },
 };
 
-const defaultTeamColors = ["#89bfd3", "#7a1319", "#07364f"];
+const defaultTeamColors = [
```

Comment:
Switch back to just one color. Multiple colors makes the demo look nicer, but any real use of it will manually specify the colors, so adding extra default colors just increases the bundle size.